pkg/platform/telemetry/telemetry.go

```diff
@@ -118,66 +121,58 @@ func (tel *Telemetry) RecordedMock(mockType string) {
 }
 
 func (tel *Telemetry) SendTelemetry(eventType string, output ...map[string]interface{}) {
+	go func() {
+		tel.mutex.Lock()
```

Comment:
Implementing go routine and mutex will cancel out each other right. I mean go routine will let the function execute them in parallel but mutex will make the function execute synchronously right.

Reply:
Sure, let me review this function again.